src/status_im2/subs/shell.cljs

```diff
@@ -70,15 +83,15 @@
                            :profile-picture (when profile-picture
                                               (str profile-picture "&addRing=0"))}
      :customization-color (or (:customization-color contact) :primary)
-     :content             (get-card-content chat communities)
+     :content             (get-card-content chat communities false nil)
      :id                  id}))
 
 (defn private-group-chat-card
-  [chat id communities]
+  [chat id communities primary-name]
   {:title               (:chat-name chat)
    :avatar-params       {}
    :customization-color (or (:customization-color chat) :primary)
-   :content             (get-card-content chat communities)
+   :content             (get-card-content chat communities true primary-name)
```

Comment:
The true boolean would be more descriptive if the function used a map for it's params.